ts/main.ts

```diff
@@ -1 +1,55 @@
 /* global data */
+
+interface FormElements extends HTMLFormControlsCollection {
+  photoUrl: HTMLInputElement;
+  title: HTMLInputElement;
+  notes: HTMLTextAreaElement;
+}
+
+const $image = document.getElementsByTagName('img');
+const imf = $image[0] as HTMLImageElement;
+const $newImg = document.getElementById('photo-url') as HTMLInputElement;
+const $form = document.querySelector('#form') as HTMLFormElement;
+
+function imgChanger(event: Event): void {
+  const $change = event.target as HTMLInputElement;
+  const picture = $change.value;
+  if (urlChecker(picture) === true) {
```

Comment:
Line 17 you don't need to check if strictly equal to true, you can just pass the variable through and it would evaluate to truthy or falsy 👍 Minor optimization